LondonBritishLibrary/orient/BLorient8824.xml

```diff
@@ -0,0 +1,418 @@
+                            <msItem xml:id="ms_i1.39">
+                                <locus from="140va"/>
+                                <title ref="LIT6428MJJudas"/>
+                                <note>Numbered as Miracle 36</note>
+                            </msItem>
+                            <msItem xml:id="ms_i1.40">
+                                <locus from="143ra"/>
+                                <title ref="LIT6429MJResurrection"/>
+                                <note>Numbered as Miracle 37, erroneously written as 17th</note>
+                            </msItem>
+                            <msItem xml:id="ms_i1.41">
+                                <locus from="153vb"/>
+                                <title ref="LIT6430MJJosephArimathea"/>
+                                <note>Numbered as Miracle 38</note>
+                            </msItem>
+                            <msItem xml:id="ms_i1.42">
+                                <locus from="159rb"/>
+                                <title ref="LIT6021MJApparitions#Apparition4">The appearance of Jesus to His Apostles on the Sea of Tiberias. 
+                                    The miraculous draught of fishes. The appearance to Peter and to John.</title>
+                                <note>Numbered as Miracle 39</note>
+                            </msItem>
+                            <msItem xml:id="ms_i1.43">
+                                <locus from="162ra"/>
+                                <title ref="LIT6021MJApparitions#Apparition5">Jesus' appearance in the cenacle. The Apostle Thomas sees and 
+                                    believes.</title>
+                                <note>Numbered as Miracle 40</note>
+                            </msItem>
+                            <msItem xml:id="ms_i1.44">
+                                <locus from="164va"/>
+                                <title ref="LIT6021MJApparitions#Apparition6">The Ascension</title>
+                                <note>Numbered as Miracle 41</note>
+                            </msItem>
+                            <msItem xml:id="ms_i1.45">
+                                <locus from="173rb"/>
+                                <title ref="LIT6401MJChildJesus"/>
+                                <note>Numbered as Miracle 42</note>
+                            </msItem>
+                        </msItem> 
+                    </msContents>
+                    <physDesc>
+                        <objectDesc form="Codex">
+                            <supportDesc>
+                                <support>
+                                    <material key="parchment"/>
+                                </support>
+                                <extent>
+                                    <measure unit="leaf">1+179+1</measure>
+                                    <measure unit="leaf" type="blank" corresp="#Iv #1r #83v #125v #178v #II">6</measure>
```

Comment:
unit="page"?

Reply:
Strelcyn regularly reports it with ff. (179 ff. = 179 folios).